src/app/components/sectioned-form/index.html

```diff
@@ -11,16 +11,30 @@
   </sky-demo-page-summary>
 
   <sky-demo-page-properties sectionHeading="Sectioned form section settings">
+    <sky-demo-page-property
+      propertyName="heading"
+    >
+    Specifies the section header.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
     Indicates whether the section is active when the form loads.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="heading"
+      propertyName="ariaInvalid"
+      isOptional="true"
+      default="false"
     >
-    Specifies the section header.
+    Indicates whether the values in the tab's content are invalid when the tabset is part of a form. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="ariaRequired"
+      isOptional="true"
+      default="false"
+    >
+    Indicates whether the values in the tabset are required when the tabset is part of a form. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="itemCount"
```

Comment:
In the description for this property, add "This property accepts `number` values."